src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -73,6 +73,9 @@ public class WalletManager : IWalletManager
         /// <summary>The broadcast manager.</summary>
         private readonly IBroadcasterManager broadcasterManager;
 
+        /// <summary>The transaction broadcast entry.</summary>
+        private TransactionBroadcastEntry transactionBroadcastEntry;
```

Comment:
I think we wouldn't need this with my suggestion above.

Reply:
Cool.